src/contents.ts

```diff
@@ -502,11 +527,26 @@ export class GitHubDrive implements Contents.IDrive {
       if (result === true) {
         return proxiedApiRequest<T>(apiPath, this._serverSettings);
       } else {
-        return browserApiRequest<T>(apiPath);
+        let apiUrl;
+        if (this.baseUrl === DEFAULT_GITHUB_BASE_URL) {
+          apiUrl = DEFAULT_GITHUB_API_URL;
+        } else {
+          apiUrl = URLExt.join(this.baseUrl, '/api/v3');
```

Comment:
This block automatically creates the correct `apiUrl` given the GitHub `baseUrl`. Since the `baseUrl` is configurable in the JupyterLab app this should address #62